KitePrintSDK/src/main/java/ly/kite/checkout/PaymentActivity.java

```diff
@@ -200,7 +251,7 @@ public void onClick(DialogInterface dialogInterface, int itemIndex) {
 
     private void payWithNewCard() {
         Intent scanIntent = new Intent(this, CardIOActivity.class);
-        scanIntent.putExtra(CardIOActivity.EXTRA_APP_TOKEN, CARD_IO_TOKEN);
+        //scanIntent.putExtra(CardIOActivity.EXTRA_APP_TOKEN, CARD_IO_TOKEN);
```

Comment:
redundant comment -- I guess as it's working card.io lost the need for a token when they were absorbed by PayPal